docs/source/serving/openai_compatible_server.md

```diff
@@ -35,8 +35,8 @@ We currently support the following OpenAI APIs:
   - [Vision](https://platform.openai.com/docs/guides/vision)-related parameters are supported; see [Multimodal Inputs](../usage/multimodal_inputs.rst).
     - *Note: `image_url.detail` parameter is not supported.*
   - We support two audio content types.
+    - Support `input_audio` content type as defined [here](https://platform.openai.com/docs/guides/audio?audio-generation-quickstart-example=audio-in).
     - Support `audio_url` content type for audio files. Refer to [here](https://github.com/vllm-project/vllm/tree/main/vllm/entrypoints/chat_utils.py#L51) for the exact schema.
```

Comment:
```suggestion
  - For image data, you can follow their [Vision guide](https://platform.openai.com/docs/guides/vision).
    - *Note: `image_url.detail` parameter is not supported.*
  - For audio data, you can follow their [Audio guide](https://platform.openai.com/docs/guides/audio?audio-generation-quickstart-example=audio-in)
    - *Note: We don't support audio generation yet.*
  - See [Multimodal Inputs](../usage/multimodal_inputs.rst) for more ways to input multimodal data.
```

Let's move `audio_url` usage to the Multimodal Inputs page.

Reply:
Multimodel input page refer to this example file (https://docs.vllm.ai/en/latest/usage/multimodal_inputs.html#id5)
do you mean add the code example to that page instead of reference link?

Reply:
I mean that you should link the usage of `audio_url` to Multimodal Inputs page. Moreover, you should add an example of `input_audio` usage to Multimodal Inputs.

Reply:
Got it. Will add sample code of audio in the multimodel input page